be/src/exprs/vectorized/array_functions.tpp

```diff
@@ -101,4 +102,127 @@ private:
         dest_offsets.emplace_back(dest_offsets.back() + hash_set->size() + has_null);
     }
 };
+
+template <PrimitiveType PT>
+class ArraySort {
+public:
+    using ColumnType = RunTimeColumnType<PT>;
+
+    static ColumnPtr process(FunctionContext* ctx, const Columns& columns) {
+        DCHECK_EQ(columns.size(), 1);
+
+        size_t chunk_size = columns[0]->size();
+        // TODO: For fixed-length types, you can operate directly on the original column without using sort index,
+        //  which will be optimized later
+        std::vector<uint32_t> sort_index;
+        ColumnPtr src_column = ColumnHelper::unpack_and_duplicate_const_column(chunk_size, columns[0]);
```

Comment:
We need to deal with null literal.
null literal will unpack to Nullable(UInt8Column). which may cause a crash in Debug